content/index.md

```diff
@@ -1,43 +1,70 @@
 type: page
 slug: index
 url: index.html
-title: Home
+title: PyCon UK 2016
 callout_big_1: PyCon UK is your community-led
 callout_big_2: volunteer-run conference
 callout_small: Cardiff City Hall from Thursday 15th to Monday 19th September 2016
 ---
 
 PyCon UK is the annual gathering of the UK Python community and its friends
 from around the world.  This year, the conference is held at [Cardiff City
-Hall](http://www.cardiffcityhall.com/), from Friday 16th to Monday 19th
-September, with a day of introductory workshops on Thursday 15th.
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+## Overview of the programme
```

Comment:
If they are too big, the CSS needs to change, not the level of headings. ``h1`` is reserved in the header, and the next level should be ``h2``. Semantics!